src/assets/locales/en.json

```diff
@@ -346,59 +346,21 @@
   "settings.row-profile": "You",
   "settings.row-room": "Room",
   "settings.row-tools": "Tools",
-  "tips.mobile.video_share_mode": "Tap to stop sharing video.",
```

Comment:
We'll need to remove these tips from the alternative translations too. Perhaps the easiest way is to update sync_locales.js so that it removes extra entries. (Currently it only adds missing entries). 
https://github.com/mozilla/hubs/blob/redesign/tips/src/assets/locales/sync_locales.js